techniques/failures/F111.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" lang="en">
+	<head>
+		<title>Failure of Success Criterion 2.5.3 due to a control with visible label text but no accessible name</title>
+		<link rel="stylesheet" type="text/css" href="../../css/editors.css"/>
+	</head>
+	<body>
+		<h1>Failure due to a control with visible label text but no accessible name</h1>
+		<section id="meta">
+			<h2>Metadata</h2>
+			<p id="id"></p>
+			<p id="technology">general</p>
+			<p id="type">failure</p>
+		</section>
+
+		<section id="applicability">
+			<h2>When to Use</h2>
+			<p>All technologies that include interactive controls (such as links or form inputs).</p>
+		</section>
+
+		<section id="description">
+			<h2>Description</h2>
+			<p>The objective of this Failure is to describe situations where speech input users cannot reliably speak the name of a control because although it has a visible label, it lacks an accessible name.</p>
+			<p>When speech input users interact with a web page, they usually speak a command followed by the reference to some visible label (like the text in a button or a link, or the text labelling some input). If the control lacks an accessible name, speech users won't be able to activate the control using its visible label.</p>	
+		</section>
+		
+		<section id="examples">
+			<h2>Examples</h2>
+      
+      <section class="example">
+				<h3>A text input with a visible label, but without an accessible name</h3>
+				<p>The text input is preceded by a visible text label "Enter name", but the text is not marked up as a <code>&lt;label&gt;</code> for the input, and there is no alternative way (e.g. <code>aria-label</code>) to provide the input with an accessible name at all.</p>
+				<pre><code>
+					&lt;p&gt;Enter name&lt;/p&gt;
+					&lt;input type="text"&gt;
+        </code></pre>
+      </section>
+
+			<section class="example">
+				<h3>A text input with a visible label and aria-labelledby pointing to a non-existent id</h3>
+				<p>The text input is preceded by a visible text label "Enter name". The text's container has an <code>id</code> of <code>nameEntry</code>, but the input has an <code>aria-labelledby</code> referencing a non-existent <code>name-entry</code>. As a result, the input lacks an accessible name altogether.</p>
+				<pre><code>
+					&lt;p id="nameEntry"&gt;Enter name&lt;/p&gt;
+					&lt;input type="text" aria-labelledby="name-entry" &gt;
+        </code></pre>
+      </section>
+    </section>
+    
+		<section id="tests">
+			<h2>Tests</h2>
+			<section class="test-procedure">
+				<h3>Procedure</h3>
+				<p>For all controls with a visible label (e.g., link text, button text, programmatically linked label, images in links with text), check that:</p>
+				<ol>
+					<li>The control has a visible text label.</li>
+					<li>The control has an accessible name.</li>
+          <li>The accessible name contains a match for the string of the visible label.</li>
```

Comment:
Since "match" can mean a few different things, I made this more closely align to the normative text

```suggestion
          <li>The accessible name contains the text string that appears as the visible lablel.</li>
```